lib/init.php

```diff
@@ -177,5 +187,4 @@ function register_site_icon_url( $response ) {
 
 add_filter( 'rest_index', 'register_site_icon_url' );
 
-add_theme_support( 'widgets-block-editor' );
```

Comment:
The removal of this seems to be why the plugin widget screen isn't working in 5.7.

Reply:
🤦‍♂️ doh. Thanks for catching. Fixed in 6d1ada7be0.